test/decorator/UseAspect.test.ts

```diff
@@ -42,8 +42,18 @@ describe('UseAspect', () => {
         sample.getSampleId();
 
         expect(beforeAspect.execute).toHaveBeenCalledTimes(1);
+
+        const expectedCtx: AspectContext = {
+            target: sample,
+            methodName: 'getSampleId',
+            functionParams: [],
+            returnValue: 1,
```

Comment:
Any idea why this has a value? My assumption would have been this would be null since it's a beforeAspect?

Reply:
Good question. It should be `null` and debugging it clearly shows that it works as we expect. Therefore, it is something related to `jest`, I think. 

I think `jest` saves a reference to the arguments passed in to the method. Since the `AspectContext` is later changed by calling the actual `getSampleId`, jest thinks that the "actual" context had a returnValue of 1. This value is set in line 38 via the constructor.